xcodeproj/internal/automatic_target_info.bzl

```diff
@@ -196,6 +202,12 @@ def calculate_automatic_target_info(ctx, build_mode, target):
         infoplists = _INFOPLISTS_ATTRS
         provisioning_profile = "provisioning_profile"
         xcode_targets = _TEST_BUNDLE_XCODE_TARGETS
+    elif _is_static_framework(target):
```

Comment:
Should this only be done for rules_ios targets?

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/issues/105 is not specific to `rules_ios` IIUC correct? If that's the case I wouldn't do this only for `rules_ios`